examples/housing_market_research/investor_analytics/portfolio_analysis.ipynb

```diff
@@ -0,0 +1,864 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After installing the required libraries, we need to load them and instantiate the Parcl Labs client. The client is a Python library designed to facilitate and optimize the user experience with the Parcl Labs API. It takes care of searching, retrieving, and formatting the data for us. To use the client, you need to have an `API_KEY` available in your [dashboard](https://dashboard.parcllabs.com/). While you can simply enter your `API_KEY` directly, it is recommended that you save it as an environment variable to make it more secure. If you are using Colab, you can follow these [steps](https://medium.com/@parthdasawant/how-to-use-secrets-in-google-colab-450c38e3ec75).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Environment setup\n",
+    "import os\n",
+    "from io import BytesIO\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib.dates as mdates\n",
+    "import plotly\n",
+    "from PIL import Image\n",
+    "import requests as re\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import plotly.express as px\n",
+    "from datetime import datetime\n",
+    "import plotly.graph_objects as go\n",
+    "from parcllabs import ParclLabsClient\n",
+    "from parcllabs.beta.charting.utils import create_labs_logo_dict\n",
+    "from parcllabs.beta.charting.styling import default_style_config as style_config\n",
+    "from parcllabs import ParclLabsClient\n",
+    "\n",
+    "# Create a Parcl Labs client\n",
+    "client = ParclLabsClient(\n",
+    "    api_key=os.environ.get('PARCL_LABS_API_KEY', \"<your Parcl Labs API key if not set as environment variable>\"), \n",
+    "    limit=10 # set default limit\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are going to retrieve the concentration of single-family homes by investor portfolio size, which can be accessed using the `portfolio_metrics.sf_housing_stock_ownership` method. Alternatively, you can access this data via the API using this [endpoint](https://docs.parcllabs.com/reference/sf_housing_stock_ownership_v1_portfolio_metrics__parcl_id__sf_housing_stock_ownership_get-1). This analysis will help us understand the composition of the SFR market in the Milwaukee metro area. We will examine investor concentration as of July 2024. The `portfolio_metrics.sf_housing_stock_ownership` method accepts `parcl_ids`, `start_date`, and `end_date` as parameters, so we need to define them in a dictionary that we can pass to the client."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Focus on two of the largest investors\n",
+    "housing_stock_ownership = {\n",
+    "    'parcl_ids': [2900134],  # Replace with appropriate Parcl IDs if needed\n",
```

Comment:
I would include the search method above this to show folks how you got to this Parcl ID. One cell above called search for Milwaukee metro with the proper parameters to get Milwaukee metro.